### ISSUE TITLE
Can you show a Table Aggregating the Pass/Fails?

### ISSUE THREAD

**Issue by frastlin** [Malvoz/web-maps-wcag-evaluation#11]
Hello,
I'm wondering if you could please have a table that counts the number of pass/fails and compares them to the total? Along the top there could be: WCAG A pass, WCAG AA pass, WCAG AAA pass, Total Pass.
Then the first row would be total number of passes, followed by the maps. That way, we can view the overall extent of the problem.
Thanks,

**Comment by Malvoz** [comment]
Thanks for the suggestion Brandon. I've added a [conformance results summary](https://github.com/Malvoz/web-maps-wcag-evaluation#conformance-results-summary) table that conveys the number of <em>failed</em> success criteria as well as the number of <em>applicable</em> success criteria (because they're not all applicable to every web map) for the different levels, and in total.

Let me know if that works for you, or how I can improve it.

**Comment by frastlin** [comment]
Exactly what I was looking for, thank you!
